src/index.js

```diff
@@ -1 +1,150 @@
-document.addEventListener("DOMContentLoaded", () => {});
+document.addEventListener("DOMContentLoaded", () => {
+    const canvas = document.getElementById('game');
+    const ctx = canvas.getContext('2d');
+
+    //Variables
+    let score;
+    let highscore;
+    let player;
+    let gravity;
+    let obstacles; //not yet used
+    let gameSpeed;
+    let keys = {};
+
+    //Event Listener for keyboard interaction
+    document.addEventListener('keydown', function (evt) {
+        keys[evt.code] = true;
+        //test for key code
+        console.log(evt.code);
+    });
+    document.addEventListener('keyup', function (evt) {
+        keys[evt.code] = false;
+    });
+
+    class Player {
```

Comment:
similar to the other comment on separation of concerns, let's dedicate a file for each class